controller/src/main/java/io/pravega/controller/server/eventProcessor/ConcurrentEventProcessor.java

```diff
@@ -40,6 +47,23 @@
 @Slf4j
 public class ConcurrentEventProcessor<R extends ControllerEvent, H extends RequestHandler<R>>
         extends EventProcessor<R> {
+    private static final long RETRY_INITIAL_DELAY = 100;
```

Comment:
Please add a utility method to retry indefinitely. These retry config values are defined at several places in the code. It is good to consolidate them at one place.